hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/OzoneClientUtils.java

```diff
@@ -212,15 +213,15 @@ public static FileChecksum getFileChecksumWithCombineMode(OzoneVolume volume,
         .setRefreshPipeline(true).setSortDatanodesInPipeline(true)
         .setLatestVersionLocation(true).build();
     OmKeyInfo keyInfo = rpcClient.getOzoneManagerClient().lookupKey(keyArgs);
-
-    // TODO: return null util ECFileChecksum is implemented.
+    BaseFileChecksumHelper helper;
     if (keyInfo.getReplicationConfig()
         .getReplicationType() == HddsProtos.ReplicationType.EC) {
-      return null;
```

Comment:
You could also create small checkSumHelper factory and create the respective instance based on replicationType.
Ex: BaseFileChecksumHelper helper = checksumHelperFactory.getHelper(keyInfo.getReplicationConfig()
        .getReplicationType())

Reply:
Yes sounds good, will add that as well with the other changes.